src/cml.js

```diff
@@ -156,6 +157,10 @@ class CML {
     return branch || (await exec(`git branch --show-current`));
   }
 
+  getDriver() {
+    return getDriver(this);
+  }
```

Comment:
should this be consolidated?

Reply:
for additional context. https://github.com/iterative/cml/blob/c9d89a44be512c9acd101aa40a91e933baa9dde2/src/cml.js#L79-L88

Reply:
Consolidated ;)